frontend/src/components/DemoModal/DemoModal.tsx

```diff
@@ -24,6 +23,9 @@ export const DemoModal = () => {
 		if (!(await formStore.validate())) {
 			return
 		}
+		if (email.indexOf('@gmail.') !== -1) {
+			return
```

Comment:
Not blocking, but maybe we create a card to give the user an error for feedback in this case?

Reply:
will update with an error tag at the bottom of the card.